apps/web/src/pages/templates/workflow/WorkflowEditor.tsx

```diff
@@ -293,7 +293,7 @@ const WorkflowEditor = () => {
                       }}
                       data-test-id="get-snippet-btn"
                     >
-                      {tagsIncludesOnboarding ? 'Test Notification Now' : 'Get Snippet'}
+                      {tagsIncludesOnboarding ? 'Trigger Notification' : 'Get Snippet'}
```

Comment:
I actually think we can change the text for all users here. Not only for onboarding. Wdyt?